.github/workflows/doc-lint-test.yaml

```diff
@@ -0,0 +1,346 @@
+#     runs-on: ${{ vars.RUNNER }}
+#     needs: build_doclinttest_image
+#     container:
+#       image: ${{ needs.build_doclinttest_image.outputs.IMAGE_NAME }}:${{ github.sha }}
+#       credentials:
+#         username: ${{ secrets.GHCR_USER }}
+#         password: ${{ secrets.GHCR_TOKEN }}
+#     steps:
+#       - name: Checkout code
+#         uses: actions/checkout@v4
+#
+#       - name: Configure Git safe directory
+#         run: git config --global --add safe.directory $PWD
+#
+#       - name: Install geoips documentation packages
+#         run: pip install -e .[doc]
+#
+#       - name: Run build pdf
+#         run: |
+#           echo "Build pdf docs"
+#           # Ie, get 'recenter_tc' out of 'NRLMMD-GEOIPS/recenter_tc'
+#           # using standard github actions variables
+#           curr_repo=`echo "$GITHUB_REPOSITORY" | sed "s,$GITHUB_REPOSITORY_OWNER/,,"`
+#           # Call build_docs.sh from the geoips default branch
+#           # Pass in
+#           #   the path to the current plugin repo,
+#           #   the name of the current repo,
+#           #   "pdf_only" to indicate producing only pdf output, and
+#           #   the path to the docs template directory in the geoips default branch.
+#           ./docs/build_docs.sh . $curr_repo pdf_only ./docs
+#           ret=$?
+#           if [[ "${ret##*:}" != *"0"* ]]; then
+#             echo "::error::Building pdf docs ${ret##*:}"
+#             exit 1
+#           fi
+
+#################################################
+# Linting
+#################################################
+  lint_bandit:
+    runs-on: ${{ vars.RUNNER }}
+    needs: build_doclinttest_image
+    container:
+      image: ${{ needs.build_doclinttest_image.outputs.IMAGE_NAME }}:${{ github.sha }}
+      credentials:
+        username: ${{ secrets.GHCR_USER }}
+        password: ${{ secrets.GHCR_TOKEN }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Run bandit
+        shell: bash -l {0}
+        run: |
+          echo "::group::bandit_analysis"
+          echo "BANDIT analysis of code"
+          ./tests/utils/check_code.sh bandit ./
+          ret=$?
+          echo "Return code: ${ret}"
+          echo "::endgroup::"
+          if [[ "${ret}" != *"0"* ]]; then
+            echo "::error::due to bandit violations, return code ${ret}"
+            exit 1
+          fi
+
+  lint_doc8:
+    runs-on: ${{ vars.RUNNER }}
+    needs: build_doclinttest_image
+    container:
+      image: ${{ needs.build_doclinttest_image.outputs.IMAGE_NAME }}:${{ github.sha }}
+      credentials:
+        username: ${{ secrets.GHCR_USER }}
+        password: ${{ secrets.GHCR_TOKEN }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Run doc8
+        shell: bash -l {0}
+        run: |
+          echo "::group::doc8_analysis"
+          echo "Doc8 analysis of code"
+          doc8 --max-line-length=120 ./docs/source/new-docs/
```

Comment:
I think that we decided that the documentation should be longer because it can contain code snippets embedded in indented syntax. So, since the code is limited to 88 characters, we need something longer for the documentation.

We could definitely add this to check_code.sh. Should we make an issue for that?